SimplenoteUITests/Generic.swift

```diff
@@ -63,6 +63,11 @@ func populateNoteList(title: String, numberToCreate: Int) {
     }
 }
 
+func getRandonId() -> String.SubSequence {
+    let id = NSDate().timeIntervalSince1970.description.suffix(6)
+    return id
+}
```

Comment:
Yeah, returning String was my first attempt but I've got the `Cannot convert return expression of type 'String.SubSequence' (aka 'Substring') to return type 'String'` message from Xcode and just set the return type to `String.SubSequence` instead of just converting variable. Returning `String` is definitely better.